packages/orbit-components/src/utils/rtl/README.md

```diff
@@ -12,133 +12,10 @@ import { OrbitProvider, defaultTheme } from "@kiwicom/orbit-components";
 </OrbitProvider>;
 ```
 
-It's also important to pass some `orbit` object, in our case `defaultTheme`.
-With this approach, all `orbit-components` will be ready to use with some right to left language.
+In addition, you also need to set the `dir` property on the `html` element to have value `rtl`.
 
-> In addition, you need to set up `direction` on `html` to have value `rtl`.
-
-## Utilities
-
-`styled-components` doesn't know automatically how to switch `margin`, `padding` and i.e. `left` properties to the other direction. So it's necessary to use some `utilities` inside your styles.
```

Comment:
How does Tailwind do it? Does it rely on logical properties?

Reply:
Yes. Tailwind has builtin classes that handle it, eg: `ms-` will apply `margin-inline-start`, that browsers translate to left margin in LTR and right margin in RTL.

One can also use the `rtl:` prefix in classes to apply styles that are only applicable in RTL

Reply:
Would it make sense to include this info in the migration guide, just to point people somewhere? I'm sure I wouldn't be the only one asking that question :)